.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "grpc-sys/grpc"]
 	path = grpc-sys/grpc
-	url = https://github.com/pingcap/grpc.git
+	url = https://github.com/grpc/grpc.git
```

Comment:
Have you checked that all patches we required are merged into the official repo?

Reply:
We are missing [Add missing Rust scenario](https://github.com/pingcap/grpc/commit/591275c0b68bb908259824a89ba2e51fdeac617a) but that is only used for benchmarking and is usually needed in a top-level grpc directory, not the submodule.

We are also missing https://github.com/grpc/grpc/commit/9099e03902a08e1209c1cc073100e049ef84a11b, which is another benchmarking thing. Again, I think we don't need it in the grpc-rs submodule, and I don't know whether we still need this at all

Reply:
> usually needed in a top-level grpc directory, not the submodule

Now how can we benchmark Rust? Move the missing Rust scenario into grpc-rs directly?

Reply:
It would be nice if we could upstream it, I have no idea if they would accept the PR.

The current benchmarking steps are to pull pingcap/grpc and pingcap/grpc-rs into sibling directories and run the benchmarks from the grpc directory. That will still work (albeit using older benchmark code).

Reply:
I think pingcap/grpc has to be updated to v1.24.x too otherwise benchmark may not work as expected. On the other hand, I think keeping our own fork and relying on our own branch can make it more flexible for us like getting bugfix merged more quickly.